### PR TITLE
AWS env update

### PR DESCRIPTION
**Pull request by Omar-Khawaja**
Updating configurations and AWS image

Consul:   1.3.1   ---> 1.4.3
Vault:      0.11.4 ---> 1.0.3
Nomad:  0.8.6  ---> 0.8.7

Installed + configured Consul Template on nodes (0.20.0) as well.

### CODE REVIEW

File: terraform/shared/scripts/setup.sh [hunk 1]
```diff
@@ -9,21 +9,26 @@ cd /ops
...
 VAULTCONFIGDIR=/etc/vault.d
 VAULTDIR=/opt/vault
 
-NOMADVERSION=0.8.6
+NOMADVERSION=0.8.7
 NOMADDOWNLOAD=https://releases.hashicorp.com/nomad/${NOMADVERSION}/nomad_${NOMADVERSION}_linux_amd64.zip
 NOMADCONFIGDIR=/etc/nomad.d
 NOMADDIR=/opt/nomad
 
+CONSULTEMPLATEVERSION=0.20.0
```
Comment (notnoop) [inline terraform/shared/scripts/setup.sh]: These versions might be better consolidated together on top of file or in a variables files, imo.

Comment (Omar-Khawaja) [reply terraform/shared/scripts/setup.sh]: Good idea. Will probably submit a separate PR soon resolving that.

**Comment by Omar-Khawaja** [comment]
@notnoop the dedicated consult template service is for securing the Nomad cluster itself by integrating with Vault's PKI secrets engine and generating TLS certs for the nodes.

**Comment by notnoop** [comment]
That's super awesome!  We should write this up as a guide or a blog if we haven't already.

**Comment by Omar-Khawaja** [comment]
> That's super awesome! We should write this up as a guide or a blog if we haven't already.

In the process of doing that 😉

**Pull request merged**